frontend/public/components/chargeback.tsx

```diff
@@ -23,7 +23,8 @@ const reportPages=[
   {name: 'Generation Queries', href: ReportGenerationQueryReference},
 ];
 
-const menuActions = [Cog.factory.ModifyLabels, Cog.factory.ModifyAnnotations, Cog.factory.Edit, Cog.factory.Delete];
+const menuActions = Cog.factory.common;
```

Comment:
so the motivation was to have declaration compact across the codebase and since I havent found any modifications declared `menuActions` const. I've just referenced the actions.
But don't have any strong opinion here.